packages/rich-text-structure/src/is-empty.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Checks if a record or record value is empty or not.
+ *
+ * @param {Object} record Record to use.
+ *
+ * @return {boolean} True if the record is empty, false if not.
+ */
+export function isEmpty( { text, formats } ) {
+	return text.length === 0 && formats.length === 0;
```

Comment:
hmm…if the text length is empty why should we have formats?

Reply:
Because there is a case where `formats.length = text.length + 1` if there is an "object" format inserted at the very end. If there is no text, there can be one trailing format entry, so we need to make sure both are zero. I should probably leave some comments about this. 😊

Reply:
what is an "object" format and where is it documented?